docs/ios-http-instrumentation.md

```diff
@@ -42,17 +42,53 @@ In your application or framework targets, add `ScopeAgent.framework` located in
 
 <!--END_DOCUSAURUS_CODE_TABS-->
 
+### Using the thin client library
+
+This same functionality is provided by `ScopeAgentClient`. With this approach, you link your main application with `ScopeAgentClient` and the tests target
+with `ScopeAgent`.
+
+Install this dependency to your test targets by adding:
+
+<!--DOCUSAURUS_CODE_TABS-->
+<!--Cocoapods-->
+
+```
+target 'MyTests' do
+ pod 'ScopeAgentClient'
+end
+```
+
+<!--Carthage-->
+
+```
+binary "https://releases.undefinedlabs.com/scope/agents/ios/ScopeAgentClient.json"
+```
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## 
+
 
 ## Usage
 
-Use the `SAURLSessionObserver.adapt(_:)` method to modify your `URLRequest` objects. The interface of `SAURLSessionObserver.adapt(_:)` is as follows:
+If you use ScopeAgent framework use the `SAURLSessionObserver.adapt(_:)` method to modify your `URLRequest` objects. The interface of `SAURLSessionObserver.adapt(_:)` is as follows:
```

Comment:
I think putting the code inside a `<!--DOCUSAURUS_CODE_TABS-->` looks better

Reply:
Done